src/VisualStudio/Core/Def/TableDataSource/VisualStudioBaseDiagnosticListTable.LiveTableDataSource.cs

```diff
@@ -175,17 +172,11 @@ private object CreateAggregationKey(DiagnosticsUpdatedArgs data)
             return new AggregatedKey(documents, liveArgsId.Analyzer, liveArgsId.Kind);
         }
 
-        private void PopulateInitialData(Workspace workspace, IDiagnosticService diagnosticService)
+        private void PopulateInitialData(Workspace _1, IDiagnosticService _2)
```

Comment:
💡 Avoid placeholders for parameter names that aren't lambdas. If the parameter is not used, better to either remove it or add a documented discard on the first line of the implementation.